app/frontend/packs/webchat.js

```diff
@@ -0,0 +1,71 @@
+function Webchat () {
```

Comment:
nit: Maybe we can rename to specify this is a support queries web chat?

Reply:
I hear you! renamed to `userSupportWebchat`